webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -29,13 +41,37 @@ export default {
   },
   data() {
     return {
+      displayedNotifications: [],
       notifications: [],
     }
   },
   props: {
     placement: { type: String },
   },
+  watch: {
+    notifications: {
+      immediate: true,
+      handler(lastQueriedNotifications) {
+        if (lastQueriedNotifications && lastQueriedNotifications.length > 0) {
```

Comment:
Prefer guard clause over if statement

```suggestion
  if (!(lastQueriedNotifications && lastQueriedNotifications.length > 0)) return
```

Reply:
Done